app/views/declaration/not_eligible.scala.html

```diff
@@ -22,7 +22,7 @@
 @(appConfig: AppConfig)(implicit request: Request[_], messages: Messages)
 
 @main_template(
-title = messages("notEligible.title"),
+title = messages("notEligible.title") + " " + messages("notEligible.titleLineTwo"),
```

Comment:
I'm thinking about new message for both of it, not add with space between

Reply:
I have addressed your comment.